app/javascript/components/modals/BadgeModal.tsx

```diff
@@ -36,7 +37,9 @@ export const BadgeModal = ({
       </div>
       <div className="num-awardees text-p-base">
         <GraphicalIcon icon="students" />
-        <strong>{badge.numAwardees}</strong> members have earned this badge.
+        <strong>{badge.numAwardees}</strong>{' '}
+        {pluralize('member', badge.numAwardees)}{' '}
+        {pluralize('has', badge.numAwardees)} earned this badge.
```

Comment:
I figure that no badge will only ever go to one person, which is why I didn't do this FYI. But I'm happy with the change :)

Reply:
Well, somebody has to be the first for newer badges that we don't bulk award 🤷